static/css/s2.css

```diff
@@ -2204,6 +2204,39 @@ div.interfaceLinks-row a {
   font-family: "Taamey Frank", "adobe-garamond-pro", "Crimson Text", Georgia, "Times New Roman", serif;
   margin-bottom: -3px;
 }
+.topicImagePicture{
+  border: 1px solid #EDEDEC;
+  max-width: 100%; 
+  height: auto;
+  /* height: 292px; */
+  padding: 0 44;
+  top: 121px;
+  left: 835px;
+}
+.topicImageCaption .int-en {
+/* styleName: English System Small; */
```

Comment:
what is the comment conveying?

Reply:
Whoops, meant to remove that. Thanks for catching it!